### ISSUE TITLE
Device Details

### ISSUE THREAD

**Issue by tchmnkyz** [digitalocean/netbox#300]
It would be nice to be able to track the Memory, CPU, Disks (components) of a server. Also some kind of integration into maybe puppet and other configuration management applications would be nice also. 

If possible, it would be nice to be able to Link a Specific graph to a interface as well as add specific graphs for each device (think CPU/Memory/etc).

**Comment by jeremystretch** [comment]
There are three unrelated features described here.

> It would be nice to be able to track the Memory, CPU, Disks (components) of a server.

There are several existing feature requests around the modules system that probably cover this.

> Also some kind of integration into maybe puppet and other configuration management applications would be nice also.

This is out of scope for the foreseeable future.

> If possible, it would be nice to be able to Link a Specific graph to a interface as well as add specific graphs for each device (think CPU/Memory/etc).

Graphs can already be defined for interfaces. Module graphs might be workable but not until after the existing feature requests are implemented.
